users/nchristus/rules.mk

```diff
@@ -0,0 +1 @@
+SRC += nchristus.c
```

Comment:
Since this file has nothing in it, you can actually safely remove this line, I believe.

Reply:
Updated.